README.md

```diff
@@ -1,2 +1,38 @@
 # django-multiple-forms
 An expansion of django's FormView to handle multiple form submissions
+
+## How To Use
+#### Install the dependency
+```
+pip install django-mulitform-views
+```
+or add `django-multiform-views` to your `requirements.in` file, then run
+```
+pip compile
+pip install -r requirements.txt
+```
+
+#### Usage
+Have your view inherit from ```FormsView```.
+
+Append `multiform_key` to `helper.layout` in the form
```

Comment:
This assumes the form uses crispy forms, so you may want to add that explicitly. The important point is to render the `multiform_key` as a hidden input in the frontend form, which can be done manually or (more simply) by adding to the crispy layout, if using crispy. You can note that the key is automatically added to the form initial by the view, so only the frontend addition is needed.